SpatialGDK/Source/SpatialGDK/Public/Schema/DebugComponent.h

```diff
@@ -50,9 +50,9 @@ struct DebugComponent : Component
 		return Data;
 	}
 
-	void ApplyComponentUpdate(const Worker_ComponentUpdate& Update) override
+	void ApplyComponentUpdate(Schema_ComponentUpdate* Update)
```

Comment:
This has been changed as of miron's entity refactory PR such that this doesn't compile on linux, because it hides a virtual function of the component base class. This was what it was before, but the refactory went in in the meantime :( . I've made a simple hacky change to get it to compile in the squash PR (renamed the function to `ApplyUpdate` but this should be revisited to make it match the rest of the component implementations.

Reply:
spoke offline, PEB my CAK

Reply:
will fix in squash if it gets approved